packages/module-loader-commonjs/src/__tests__/index.test.ts

```diff
@@ -2,16 +2,23 @@
  * @jest-environment node
  */
 
+import {RequestInfo, RequestInit} from 'node-fetch';
+import ProxyAgent from 'proxy-agent';
 import {createCommonJsModuleLoader, loadCommonJsModule} from '..';
 
 let mockResponse: string;
+let requestUrl: RequestInfo;
+let requestInitOptions: RequestInit;
```

Comment:
Pleas rename to `requestInit`

Reply:
Hi @clebert , agreed, I changed, thanks!